src/cascadia/TerminalSettings/TerminalSettings.cpp

```diff
@@ -13,6 +13,7 @@ namespace winrt::Microsoft::Terminal::Settings::implementation
         _defaultForeground{ DEFAULT_FOREGROUND_WITH_ALPHA },
         _defaultBackground{ DEFAULT_BACKGROUND_WITH_ALPHA },
         _colorTable{},
+        _tabTitle{},

```

Comment:
if you remove it from the idl you can remove it from all the TerminalSettings files